docs/manual/src/types/structs.md

```diff
@@ -141,3 +108,5 @@ data class TodoEntry (
 ```
 
 This works for Swift and Python targets too.
+
+XXX - what does this even mean? :) Do proc-macros support it?
```

Comment:
This is also unclear to me. I would assume this is the default behavior of `Option`?

Reply:
I think proc-macros support this.  As to the "Swift and Python targets too", the only meaning I can figure out is that it's saying just because there's a Kotlin example, don't assume that it's Kotlin-only.   That doesn't seem like something we really need to say though.